### DISCUSSION TITLE
OrientDB v 3.2.0 released!

### DISCUSSION THREAD

**Discussion by luigidellaquila** [orientechnologies/orientdb#9595] [General]
Dear OrientDB Community,

I'm happy to announce that OrientDB v 3.2.0 is now GA and available for download!

https://orientdb.org/download

If you are curious to know what's new in v 3.2, here you can find a short list:

https://orientdb.org/docs/3.2.x/release/3.2/What-is-new-in-OrientDB-3.2.html

I would like to send a big big THANK YOU to all the people who contributed to this milestone in so many ways: with code contributions and pull requests, testing the snapshots and sending feedback, commenting on new features and possible enhancements, opening issues and providing support to other Community members, or just talking about OrientDB and spreading its name.

As you know, after the Community Edition is released, also SAP Enterprise OrientDB will follow; this will be the final release of the commercial product, but his doesn't mean that OrientDB development is stopping here of course!

OrientDB Community Edition will continue to evolve as a purely Open Source project, so this is an excellent chance for you to get involved in this process, with new ideas, feedback and code contributions.

Enjoy the new release!

Luigi

**Comment by mdre** [comment]
Hi Luigi.

This means that "SAP Enterprise OrientDB" will no longer exist? 
Version 3.3 for example will be only Community?
In that case, are there a way to contract your enterprise to get primary level support? 

Thank's! 

Marcelo

**Reply by luigidellaquila** [reply]
Hi Marcelo,

SAP will keep supporting existing customers on v 3.2 EE, you can contact orientdb_enquiries@sap.com for more info.
Next releases will be driven by the Community, so yes, from my understanding a possible next v 3.3 or 4.0 will be only a Community Edition, fully Open Source.

Thanks

Luigi

**Reply by PhantomYdn** [reply]
@luigidellaquila , I hope that SAP is not going to "shutdown" OrientDB...

**Reply by mmacfadden** [reply]
Does this also imply that the level of development / contribution by OrientDB / SAP Employees would be reduced as well?  Just wondering if this impacts the pace at which OrientDB has been developed?

**Reply by luigidellaquila** [reply]
Hi @mmacfadden 

In last years, our main goal was to make OrientDB always more stable and solid, and I think we will continue with the same pace on this goal, mainly on 3.2 hotfixes, but the codebase will benefit of this effort in general, so also for possible future releases.

What will likely change is the way OrientDB Community Edition will evolve in next releases: since it will be a pure OSS/Community effort, I expect the Community to have a much more central role on the decisions and on the contributions.

I cannot commit on any deadlines, but what I can say is that I love the project and I'll keep contributing as much as I can

Thanks

Luigi
